### PR TITLE
Update to APM docs per request from PM

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ For details on how to add a specific APM tool to your app, see one of the follow
 A summary of the useful metrics provided by Mendix is in the section [Summary of Mendix Metrics](#summary), below.
 
 {{% alert color="warning" %}}
-Monitoring using an APM tool is not supported in the deprecated Mendix Cloud v3, nor in default deployment buildpacks for other cloud platforms.
+Monitoring using an APM tool is not supported in the deprecated Mendix Cloud v3. For support on other cloud deployment options, such as Private Cloud, refer to their dedicated [documentation pages](/developerportal/deploy/private-cloud-monitor/).
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md:37]: Ravi says that no customers are on V3 any more.
@Sakshi-Dhakad Is this true?
If so, we might as well take out the information about it?

**Review by MarkvanMents — COMMENTED** [review]
Looks good.
Thanks for doing this, Kasia.
I'd like to take this opportunity to check the status of Cloud V3 so we can prioritise what to do with other V3 documentation.

File: content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md (hunk 1 above)
Comment (Sakshi-Dhakad) [reply content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md:37]: Thanks Kaisa, looks good. Does this also reflect on the child pages on monitoring, i.e. on data dog, splunk and App Dynamics?

Comment (Sakshi-Dhakad) [reply content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md:37]: On the rest of V3 documentation, that's right, all v3 apps have now been stopped / migrated. I'm double-checking with the team to make sure we can make changes to documentation overall too. I'll keep you both posted!

Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md:37]: @Sakshi-Dhakad Yes, I've updated 4 files in the PR - intro, data dog, splunk, and app dynamics. If all looks good, I'm going to merge the PR.

Comment (Sakshi-Dhakad) [reply content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md:37]: Thanks @katarzyna-koltun-mx , looks good! I missed the other files in the first look.

**Pull request merged**